polkadot/node/core/backing/src/lib.rs

```diff
@@ -992,14 +1007,18 @@ async fn construct_per_relay_parent_state<Context>(
 
 	let parent = relay_parent;
 
-	let (session_index, validators, groups, cores) = futures::try_join!(
+	let (session_index, validators, groups, cores, disabled_validators) = futures::try_join!(
 		request_session_index_for_child(parent, ctx.sender()).await,
 		request_validators(parent, ctx.sender()).await,
 		request_validator_groups(parent, ctx.sender()).await,
 		request_from_runtime(parent, ctx.sender(), |tx| {
 			RuntimeApiRequest::AvailabilityCores(tx)
 		},)
 		.await,
+		request_from_runtime(parent, ctx.sender(), |tx| {
```

Comment:
This is not good. We need to support old runtimes which hasn't got this runtime call enabled.